openapi/components/schemas/RiskScoreSimulationJob.yaml

```
@@ -0,0 +1,77 @@
+type: object
+description: Resource to simulate a risk score calculation given a set of risk score settings.
+required:
+  - rules
+  - period
+properties:
+  id:
+    type: string
+    description: ID of the job.
+    maxLength: 50
+    example: rsim_job_0YVCNQ2C1AD1RA3HXKP492GNZB
+    readOnly: true
+  status:
+    type: string
+    description: Status of the job.
+    enum:
+      - pending
+      - processing
+      - completed
+      - failed
+      - stopped
+    example: completed
+    readOnly: true
+  summary:
+    description: Summary of the simulation.
+    readOnly: true
+    type:
+      - array
+    items:
+      type: object
+      properties:
+        metric:
+          type: string
+          enum:
+            - transactionsProcessed
+            - affectedTransactions
+            - chargebackToBlockedTransactions
+            - refundedToBlockedTransactions
+            - declinedToBlockedTransactions
+            - unblockedTransactions
+            - blockedTransactions
+        count:
+          type: integer
+        amount:
+          $ref: ./Money.yaml
+  rules:
+    description: Risk score rules.
+    $ref: ./RiskScoreRules.yaml
+  period:
+    type: object
+    description: Full period of the simulation.
+    properties:
+      startDate:
+        type: string
+        format: date
+        example: 01-09-2022
+      endDate:
+        type: string
+        format: date
+        example: 30-09-2022
+  createdAt:
+    description: Date and time when the simulation job was created.
+    $ref: ./ServerTimestamp.yaml
```

```suggestion
  createdTime:
    $ref: ./CreatedTime.yaml
  updatedTime:
    $ref: ./UpdatedTime.yaml
```
To unify API definition, an example https://github.com/Rebilly/api-definitions/blob/5d7ee896bc2d37b38f3f8faf06b94d2e6dc04c43/openapi/components/schemas/Customer.yaml#L54